PowerEditor/src/Notepad_plus.rc

```diff
@@ -116,8 +116,11 @@ IDI_UNDO_DISABLE_ICON           ICON      "icons/undo_dis.ico"
 IDI_REDO_DISABLE_ICON           ICON      "icons/redo_dis.ico"
 //
 IDI_SAVED_ICON                  ICON      "icons/saved.ico"
+IDI_SAVED1_ICON                 ICON      "icons/saved1.ico"

```

Comment:
`saved2.ico` for me is more suitable:
"Shreck 2" came after "Shreck", but there's no "Shreck 1".

Reply:
It originated from the possibility of having in the future, possibly:

* Select Alternate icon set 1 <-- this current one
* Select Alternate icon set 2 <-- possible future set

But yes, no problem, I can change this.

No `c` in `Shrek` :-)